lib/bunch/math.ex

```diff
@@ -49,15 +49,34 @@ defmodule Bunch.Math do
 
   ## Examples
 
-      iex> #{inspect(__MODULE__)}.int_part(10, 4)
+      iex> #{inspect(__MODULE__)}.max_multiple_lt(4, 10)
       8
-      iex> #{inspect(__MODULE__)}.int_part(7, 7)
-      7
+      iex> #{inspect(__MODULE__)}.max_multiple_lt(2, 6)
+      6
```

Comment:
Looking at this example I realised the name is wrong, I think it should end with `lte`